ros/src/waypoint_updater/waypnt_updater.py

```diff
@@ -63,6 +63,9 @@ def get_accel_time(S, Vi, Vf):
 
 class WaypointUpdater(object):
     def __init__(self):
+        #self.testing = True
+        self.testing = False
```

Comment:
Added self testing routine for start of track 2.

Reply:
The test routine waypoints look like track 1, but it might be helpful to also have a similar test routine for track 2 with a single stop light manually specified to test the transitions at such low speeds.